src/xiaomi_ble/parser.py

```diff
@@ -971,6 +971,49 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             if self._parse_xiaomi(service_info, service_info.name, data):
                 self.last_service_info = service_info
 
+        if hhc_data := service_info.service_data.get(SERVICE_HHCCJCY10):
+            if self._parse_hhcc(service_info, service_info.name, hhc_data):
+                self.last_service_info = service_info
+
+    def _parse_hhcc(
+        self, service_info: BluetoothServiceInfo, name: str, data: bytes
+    ) -> bool:
+        """Parser for Pink version of HHCCJCY10."""
+        if len(data) != 9:
+            return False
+
+        identifier = short_address(service_info.address)
+        self.set_title(f"Plant Sensor {identifier} (HHCCJCY10)")
+        self.set_device_name(f"Plant Sensor {identifier}")
+        self.set_device_type("HHCCJCY10")
+        self.set_device_manufacturer("HHCC Plant Technology Co. Ltd")
+
+        xvalue_1 = data[0:3]
+        (moist, temp) = struct.unpack(">BH", xvalue_1)
+        self.update_predefined_sensor(SensorLibrary.TEMPERATURE__CELSIUS, temp / 10)
+        self.update_sensor(
```

Comment:
You already mentioned it that you still need to check it, but `SensorLibrary.MOISTURE__PERCENTAGE` is available.